### ISSUE TITLE
Apertium RDF - metadata per dictionary

### ISSUE THREAD

**Issue by jogracia** [acoli-repo/acoli-dicts#21]
Metadata file to be added for every dictionary. For inspiration, I copy the metadata of a dictionary in Apertium RDF v1.0:

Zenodo link could be added as a rdfs:sameAs

<rdf:RDF xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dcat="http://www.w3.org/ns/dcat#" xmlns:dct="http://purl.org/dc/terms/">
	<dcat:Dataset rdf:about="http://linguistic.linkeddata.es/set/apertium/EN-ES">
		<owl:sameAs rdf:resource="http://datahub.ckan.io/dataset/apertium-rdf-en-es"></owl:sameAs>
		<owl:sameAs rdf:resource="https://datahub.ckan.io/dataset/47e9d8cc-5da9-4c02-960e-c00abee2b0d9"></owl:sameAs>
		<dct:source rdf:resource="http://hdl.handle.net/10230/17110"></dct:source>
		<dct:language rdf:resource="http://id.loc.gov/vocabulary/iso639-1/en"></dct:language> 
		<dct:language rdf:resource="http://id.loc.gov/vocabulary/iso639-1/es"></dct:language> 
		<dct:license rdf:resource="http://purl.oclc.org/NET/rdflicense/gpl3.0"></dct:license>
		<rdfs:seeAlso rdf:resource="http://purl.org/ms-lod/UPF-MetadataRecords.ttl#Apertium-en-es_resource-5v2"></rdfs:seeAlso>
		<rdfs:seeAlso rdf:resource="http://dbpedia.org/resource/Apertium"></rdfs:seeAlso>
	</dcat:Dataset>
</rdf:RDF>

**Comment by jogracia** [comment]
another suggestion: we could add a "version" tag ( v2.2 ?)

**Comment by max-ionov** [comment]
@jogracia , should this be done automatically? If so, I foresee problems with CKAN, hdl.handle.net and METASHARE since we don't know the IDs in advance (we can generate `http://datahub.ckan.io/dataset/apertium-rdf-en-es` but not `https://datahub.ckan.io/dataset/47e9d8cc-5da9-4c02-960e-c00abee2b0d9`

**Comment by chiarcos** [comment]
This is part of the general problem of Linked-Data incompliant data hosting
policy I've been complaining about at times. In this particular case, we
may find a partial workaround by using Purl or (better) W3ID, but these
cannot be automatized either, but have to be manually set up and maintained.

Am Di., 12. Apr. 2022 um 12:54 Uhr schrieb Max Ionov <
***@***.***>:

**Comment by jogracia** [comment]
Good point. What we actually did for Apertium RDF v1.0 was to upload the data into Datahub and then consult the generated IDs and manually include the sameAs statements in the metadata files, to finally update only the metadata files (the data remained unchanged). Now Datahub is discontinued, but we can do the same with Zenodo (seeAlso) and Linghub (sameAs) once the data is documented there.

**Comment by chiarcos** [comment]
But shouldn't the metadata not be bundled with the data? If I recall
correctly, Zenodo (at least) will generate a new DOI/URI for every version,
so there is no good way to update previously deposited metadata (you need
to ask them for every single item). See first point under
https://help.zenodo.org/.
